README.md

```diff
@@ -1 +1,123 @@
-# Leaderbord_Project
\ No newline at end of file
+# Leaderbord_Project
+
+<a name="readme-top"></a>
+
+# 📗Table of Contents
+
+- [📖 About the Project](#about-project)
+- [🛠 Built With](#built-with)
+- [Tech Stack](#tech-stack)
+- [Key Features](#key-features)
+- [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Prerequisites](#prerequisites)
+  - [Setup](#setup)
+  - [Usage](#usage)
+- [👥 Authors](#authors)
+- [👥 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Leaderbord_Project] <a name="about-project"></a>
+
+**[Leaderbord_Project]** The "leaderboard" website displays scores submitted by different players. It also allows you to submit your score. All data is preserved
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li>HTML</li>
+    <li>CSS</li>
+    <li>JavaScript</li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- An HTML File
+- A CSS File
+- A Javascript file
+- ES6
+- A modules file
+- An md file
+- Webpack
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+> Coming soon!!
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+### Prerequisites
+
+In order to run this project you need:
+
+A Browser
+
+### Setup
+
+Clone this repository to your desired folder:
+
+`https://github.com/Ouail-01/Leaderboard_Project.git`
+
+### Install
+
+Install this project with:
+A commandline interface e.g Gitbash
+
+### Usage
+
+To run the project, execute the following command:
+`npm run build`
+
```

Comment:
- [OPTIONAL] As suggested by the previous code reviewer, the right commands to set up and run your project locally would be `npm install` and `npm start`, please do well to update that as this will be help to non technical users and some of them are potential recruiters.

Reply:
Thank you @Mihndim2020, I will update that.